src/Controller/ReservationController.php

```diff
@@ -199,9 +199,9 @@ public function payment(Request $request)
     /**
      * @Route("/reservation/{id}", name="reservation_delete", methods="DELETE")
      */
-    public function delete(Request $request, Reservation $reservation): Response
+    public function delete(Request $request, Reservation $reservation)
     {
-        if ($this->isCsrfTokenValid('delete'.$reservation->getId(), $request->request->get('_token'))) {
+        if ($this->isCsrfTokenValid('delete'.$reservation->getId(), $request->request->get('token'))) {
```

Comment:
nesuvokiu kam tas token? cia gi ne formos submitas. be to nera patikrinimo ar rezervacija priklauso tam uzeriui. dabar bet kas gali istrinti